west.yml

```
@@ -101,7 +101,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 33e59d44afb636c4fea4671a7ad301cbb619b7ec
+      revision: pull/304/head
```

As https://github.com/nrfconnect/sdk-nrfxlib/pull/306 has been created I think that we should go with it here.

```suggestion
      revision: pull/306/head
```